cli_tools/common/utils/param/param_utils_test.go

```diff
@@ -434,3 +434,23 @@ func TestPopulateMissingParametersPopulatesStorageLocationWithScratchBucketLocat
 	assert.Nil(t, err)
 	assert.Equal(t, "US", storageLocation)
 }
+
+func TestGetGlobalResourceName(t *testing.T) {
```

Comment:
I would split these in separate methods since they test different scenarios. If one of these fail, you won't know what exactly failed without looking at line number. With one func-one test case setup, func name will tell you immediately what failed. Applies to all test cases below.